### PR TITLE
Updated the link to licensing requirements

### PR DESCRIPTION
**Pull request by anwara-ms**
@MicrosoftHeidi - I have updated the licensing requirements link to reflect more relevant information

### CODE REVIEW

**Comment by scanum** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: Teams/security-compliance-overview.md [hunk 1]
```diff
@@ -171,8 +171,7 @@ The following figure indicates the ingestion flow of Teams Meetings and calling
 
 When it comes to information protection capabilities, Microsoft 365 subscriptions, Office 365 subscriptions, and the associated standalone licenses will determine the available feature set.
 
-For information on determining the licensing needs to implement features for security and compliance, please review [Licensing for Office 365 or Microsoft 365](https://download.microsoft.com/download/8/7/7/877B1713-671E-43AA-BB79-AF8478C64AFF/Licensing-Microsoft-365.pdf).
-
+For information on determining the licensing needs to implement features for security and compliance, please review the [licensing requirements](https://docs.microsoft.com/en-us/office365/servicedescriptions/microsoft-365-service-descriptions/microsoft-365-tenantlevel-services-licensing-guidance/microsoft-365-security-compliance-licensing-guidance) for security and compliance features
```
Comment (JohanFreelancer9) [inline Teams/security-compliance-overview.md:174]:
```suggestion
For information on determining the licensing needs to implement features for security and compliance, please review the [licensing requirements](https://docs.microsoft.com/office365/servicedescriptions/microsoft-365-service-descriptions/microsoft-365-tenantlevel-services-licensing-guidance/microsoft-365-security-compliance-licensing-guidance) for security and compliance features.
```

**Comment by scanum** [comment]
@MicrosoftHeidi  A PM needs help updating some PowerShell cmdlet topics for a feature release.

**Comment by scanum** [comment]
@anwara-ms PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Comment by anwara-ms** [comment]
How do I merge changes?

Thank you!
 --
Anwar Ali | PM @ TEAMS | LinkedIn<https://nam06.safelinks.protection.outlook.com/?url=https%3A%2F%2Fwww.linkedin.com%2Fin%2Fanwarali&data=04%7C01%7CAli.Anwar%40microsoft.com%7C61a24ee754db4b1493e408d6b4dc7b41%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C636895256392090042%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C-1&sdata=KpQ4z1jxZOc8s8IWXAYS3oVAbqKpT94E%2BzEB%2FLypv2c%3D&reserved=0>
IP, Security and Compliance Team

**Comment by anwara-ms** [comment]
+@Heidi Payne<mailto:heidip@microsoft.com>

Thank you!
 --
Anwar Ali | PM @ TEAMS | LinkedIn<https://nam06.safelinks.protection.outlook.com/?url=https%3A%2F%2Fwww.linkedin.com%2Fin%2Fanwarali&data=04%7C01%7CAli.Anwar%40microsoft.com%7C61a24ee754db4b1493e408d6b4dc7b41%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C636895256392090042%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C-1&sdata=KpQ4z1jxZOc8s8IWXAYS3oVAbqKpT94E%2BzEB%2FLypv2c%3D&reserved=0>
IP, Security and Compliance Team

From: Anwar Ali
Sent: Tuesday, July 7, 2020 11:02 AM
To: MicrosoftDocs/OfficeDocs-SkypeForBusiness <reply@reply.github.com>; MicrosoftDocs/OfficeDocs-SkypeForBusiness <OfficeDocs-SkypeForBusiness@noreply.github.com>
Cc: Assign <assign@noreply.github.com>
Subject: RE: [MicrosoftDocs/OfficeDocs-SkypeForBusiness] Updated the link to licensing requirements (#5032)

How do I merge changes?

Thank you!
 --
Anwar Ali | PM @ TEAMS | LinkedIn<https://nam06.safelinks.protection.outlook.com/?url=https%3A%2F%2Fwww.linkedin.com%2Fin%2Fanwarali&data=04%7C01%7CAli.Anwar%40microsoft.com%7C61a24ee754db4b1493e408d6b4dc7b41%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C636895256392090042%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C-1&sdata=KpQ4z1jxZOc8s8IWXAYS3oVAbqKpT94E%2BzEB%2FLypv2c%3D&reserved=0>
IP, Security and Compliance Team

**Pull request merged**